packages/shared/src/components/MainLayout.tsx

```diff
@@ -38,13 +42,19 @@ export default function MainLayout({
   children,
   showOnlyLogo,
   greeting,
+  activePage,
   onLogoClick,
+  onNavTabClick,
+  enableSearch,
   onShowDndClick,
 }: MainLayoutProps): ReactElement {
+  const router = useRouter();
   const { windowLoaded } = useContext(ProgressiveEnhancementContext);
   const { user, showLogin, loadingUser } = useContext(AuthContext);
   const [showGreeting, setShowGreeting] = useState(false);
 
+  const useActivePage = activePage || router?.asPath;
```

Comment:
I suggest receiving `router?.asPath` from the outside.
Let's keep this component platform-agnostic if possible.
You can utilize the existing `activePage` property to pass `router?.asPath` from the nextjs app

Reply:
Sure thing! Fair enough ☺️